src/test/java/teammates/test/cases/ui/browsertests/FeedbackRubricQuestionUiTest.java

```diff
@@ -304,6 +305,33 @@ private void testDeleteRubricQuestionAction() {
         assertNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 1));    
     }
     
+    private void testInputJsValidationForRubricQuestion() {
+        ______TS("JS validation test");
```

Comment:
Add comments to make it easier to understand what is being done in the test, and what is tested.